templates/org/aac/aac-article-title-content.md

```diff
@@ -10,132 +11,148 @@ _Download a [Visio file](https://arch-center.azureedge.net/architecture.vsdx) th
+<!--
+- Title this section "Workflow" if data isn't in the scenario.
+- Use a numbered list that describes the dataflow or workflow of each step in the solution. Use a bulleted list if there are no numbers in the diagram. Start from the user or external data source, and then follow the flow through the rest of the solution. The following section uses the previous diagram as an example and should be updated for your specific article. The following dataflow corresponds to the previous diagram:
+-->
 
-The following workflow (or dataflow) corresponds to the above diagram:
+The following workflow (or dataflow) corresponds to the previous diagram:
 
 - **Thing 1**. Explain what happens with the first technology in the diagram.
 
 - **Thing 2**. Explain what happens with the second technology in the diagram.
 
 ### Components
-
-> A bulleted list of components in the architecture (including all relevant Azure services) with links to the product service pages. This is for lead generation (what business, marketing, and PG want). It helps drive revenue.
-
-> Why is each component there?
-> What does it do and why was it necessary?
-> Link the name of the service (via embedded link) to the service's product service page. Be sure to exclude the localization part of the URL (such as "en-US/").
-
-- Examples: 
-  - [Azure App Service](https://azure.microsoft.com/services/app-service)
-  - [Azure Bot Service](https://azure.microsoft.com/services/bot-service)
+<!-- 
+- Add a bulleted list of components in the architecture. It includes all relevant Azure services and has links to the Well-Architected Framework service guide for the product. This is for lead generation (what business, marketing, and PG want). It helps drive revenue.
+- Describe why each component is necessary and what it does.
+- Link the name of the service via embedded link to the Azure Well-Architected service guide if it exists, or to the service's product page. Exclude the localization part of the URL, such as `en-us`.
+- The following section contains example components but should be updated for your specific article.
+-->
+
+- Examples:
   - [Azure Cognitive Services Language Understanding](https://azure.microsoft.com/services/cognitive-services/language-understanding-intelligent-service)
   - [Azure Cognitive Services Speech Services](https://azure.microsoft.com/services/cognitive-services/speech-services)
   - [Azure SQL Database](https://azure.microsoft.com/services/sql-database)
   - [Azure Monitor](https://azure.microsoft.com/services/monitor): Application Insights is a feature of Azure Monitor.
-  - [Resource Groups][resource-groups] is a logical container for Azure resources.  We use resource groups to organize everything related to this project in the Azure console.
+  - [Resource Groups][resource-groups] is a logical container for Azure resources. We use resource groups to organize everything related to this project in the Azure console.
```

Comment:
Does the link in line 38 work on staging? There's no link list in this article, and the link style is different from lines 34-37. Maybe it's best to just delete this line.

Reply:
Should we link to this documentation about resource groups?: https://learn.microsoft.com/en-us/azure/azure-resource-manager/management/manage-resource-groups-portal#what-is-a-resource-group